### PR TITLE
Annotations: use common processing structure

### PR DESCRIPTION
**Pull request by ryantxu**
I am mostly trying to validate that the function we would add to core makes sense here...

### CODE REVIEW

File: src/DataSource.ts [hunk 1]
```diff
@@ -48,21 +49,16 @@ export class DataSource extends DataSourceWithBackend<GitHubQuery, GithubDataSou
...
-
-    const results = view.map(item => {
-      return {
-        title: `${request.annotation.name} - ${annotation.queryType}`,
-        time: item[annotation.timeField || 'name'],
-        text: item[annotation.field || 'name'],
-      };
+    return getAnnotationsFromFrame(res.data[0], {
+      field: {
+        // title: `${request.annotation.name} - ${annotation.queryType}`,
```
Comment (ryantxu) [inline src/DataSource.ts:57]: Do you want to have the same title for everything in the query result?

Comment (kminehart) [reply src/DataSource.ts:57]: I don't see why not. After using this for a bit, the ` - ${annotation.queryType}` thing caused a lot of stuttering "Open Issues - Issues" for exxample.

**Comment by kminehart** [comment]
I checked out the branch and made sure it still worked. Looks good to me. :+1:

**Comment by ryantxu** [comment]
Great -- after this is merged into master, and we can target 7.2, this should be removed form the github project

**Pull request merged**